frontend/src/app/south/south-display/south-display.component.ts

```diff
@@ -28,10 +59,120 @@ export class SouthDisplayComponent implements OnInit {
             return this.southConnectorService.getSouthConnector(paramSouthId);
           }
           return of(null);
+        }),
+        switchMap(southConnector => {
+          if (!southConnector) {
+            return of(null);
+          }
+          this.southConnector = southConnector;
+          return this.southConnectorService.getSouthConnectorTypeManifest(this.southConnector!.type);
         })
       )
-      .subscribe(southConnector => {
-        this.southConnector = southConnector;
+      .subscribe(manifest => {
+        if (!manifest) {
+          return;
+        }
+        const rowList: Array<Array<OibFormControl>> = [];
+        // Create rows from the manifest so settings can be put together on the same row
+        manifest.settings.forEach(element => {
+          if (this.southConnector?.settings) {
+            element.currentValue = this.southConnector.settings[element.key];
+          }
+          if (element.newRow || rowList.length === 0) {
+            rowList.push([element]);
+          } else {
+            rowList[rowList.length - 1].push(element);
+          }
+        });
```

Comment:
I feel I already see this logic in a previous Pr, so maybe this can be refactored and shared?